data/json/items/book/computer.json

```diff
@@ -5,6 +5,34 @@
     "category": "manuals",
     "name": { "str": "SICP", "str_pl": "copies of SICP" },
     "description": "A classic text, \"The Structure and Interpretation of Computer Programs.\" Written with examples in LISP, but applicable to any language.",
+    "variants": [
+      {
+        "id": "SICP",
+        "name": { "str": "SICP", "str_pl": "copies of SICP" },
+        "description": "A classic text, \"The Structure and Interpretation of Computer Programs.\" Written with examples in LISP, but applicable to any language.",
+        "weight": 1
+      },
+      {
+        "id": "SICP_1",
+        "name": { "str": "Introduction to Algorithms", "str_pl": "copies of Introduction to Algorithms" },
+        "description": "Also known as CLRS, this book contain a deep explanation range of algorithms divided across several self-contained chapters, with pseudocode presentation.",
+        "weight": 1
+      },
+      {
+        "id": "SICP_2",
+        "name": { "str": "The Art of Computer Programming, Vol. 1", "str_pl": "copies of The Art of Computer Programming, Vol. 1" },
+        "description": "Huge reference  manual about fundamental algorithms, with lot of examples and math.",
+        "fun": -2,
+        "weight": 1
+      },
+      {
+        "id": "SICP_3",
+        "name": { "str": "The Art of Computer Programming, Vol. 3", "str_pl": "copies of The Art of Computer Programming, Vol. 3" },
```

Comment:
Where did volume 2 go 🤔

Reply:
Volume 2 ~~was lost~~ is about seminumerical algorithms, and there is no information about what it is that i can use in description
There is also volume 4, that i also ignored, about combinatorial algorithms